CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix escapes prematurely terminated by terminators in unicode glyphs
 - Incorrect location when clicking inside an unfocused window on macOS
 - Startup mode `Maximized` on Windows
+- Crash when trying to write a fullwidth character in the last column
```

Comment:
Should we state that it was with disabled line wrapping?

Reply:
Should be good now?